### PR TITLE
JSON Standardization

### PR DESCRIPTION
**Pull request by BryanGraeser**
This PR adapts our json files and json file handling to reflect the data serialization now described on our architecture doc: https://docs.google.com/document/d/16eRPdaEX9A8tzajSBrh408LHoDjVKKBYueRL1Z9o_WA/edit# .

Please let me know what you think of this data serialization model, particularly if there is anything in the level json that should be in the global json or vice versa.

### CODE REVIEW

File: core/assets/jsons/global.json
Comment (lsizemore8) [inline core/assets/jsons/global.json:12]: What is the walk limit?

File: core/assets/jsons/global.json
Comment (lsizemore8) [inline core/assets/jsons/global.json:58]: Why are walls both in the global json and the level json/why is the information overlapping between the two? Is it the global one is a default wall and the level one is for specialization only?

**Review by lsizemore8 — CHANGES_REQUESTED** [review]
Left a couple questions. Main action item is the fact that walls are in both the global and local json, which I don't think should occur.

**Comment by lsizemore8** [comment]
Also -> do we need to consider floors? This may be a design question, but unless we want the floor to be one color and we want no textures, we'll need a floor.

File: core/assets/jsons/global.json
Comment (BryanGraeser) [reply core/assets/jsons/global.json:58]: There isn't any overlapping information except texture. I put all information that is constant for walls in the global json, and put all dynamic data (size, pos, and texture) in the level json. 

I did put texture also in the global json thinking that could be the default texture if one is not specified in the level json, but I'm not committed to that and may room it (also the code doesn't handle that case yet.

File: core/assets/jsons/global.json
Comment (BryanGraeser) [reply core/assets/jsons/global.json:12]: This is the cooldown limit between walk animations

**Comment by BryanGraeser** [comment]
Responded to your comments. Walls are meant to be in both, just like how player and enemy are in both (global constants vs dynamic data). 

For floors, that's a good point, but I think we can wait until designs are done so we have a better idea how that will be implemented.

File: core/assets/jsons/global.json
Comment (BryanGraeser) [reply core/assets/jsons/global.json:58]: The default texture is now handled in WallModel (ie uses default if other is not specified).

Comment (lsizemore8) [reply core/assets/jsons/global.json:58]: The "pad" attribute is overlapping in some walls (the second half of the walls in the level json), and it is the same value from the global. That's what I also meant in the "overlapping" aspect (in addition to texture).

Comment (BryanGraeser) [reply core/assets/jsons/global.json:58]: Oh whoops, yeah I can fix that

**Review by lsizemore8 — APPROVED** [review]
Looks good to me. I'll approve it, but if anyone hasn't looked at it, please do before we merge it.

**Review by stallop — APPROVED** [review]
Structure makes sense to me and I don't spot any issues.

**Pull request merged**
